Dockerfile.static.in

```diff
@@ -1,6 +1,8 @@
 FROM {tag}
 
 RUN /opt/ocfweb/venv/bin/sassc /opt/ocfweb/ocfweb/static/scss/site.scss /opt/ocfweb/ocfweb/static/scss/site.scss.css
+RUN /opt/ocfweb/recursive-minify css /opt/ocfweb/ocfweb/
+RUN /opt/ocfweb/recursive-minify js /opt/ocfweb/ocfweb/
```

Comment:
i'd be tempted to drop the script and just write (untested, but probably works)

```dockerfile
RUN find \( -name '*.js' -o -name '*.css' \) -exec yui-compressor -o {} {} \;
```

/shrug

Reply:
I tested it and it does work. Thank you for the suggestion!